src/widgets/icons.js

```diff
@@ -126,3 +126,13 @@ export const WifiIcon = ({ color, size, style }) => (
 );
 WifiIcon.defaultProps = { color: DARK_GREY, size: 22, style: { marginLeft: 10 } };
 WifiIcon.propTypes = { color: PropTypes.string, size: PropTypes.number, style: PropTypes.object };
+
+export const TemperatureIcon = ({ color, size, style }) => (
```

Comment:
Pure 😆 ?

Reply:
pretty much all the icons are non-memoized - it should be consistent though!